src/buffer/scroll_view.rs

```diff
@@ -408,24 +408,41 @@ pub fn view<'a>(
         }
     };
 
-    let divider = if show_backlog_divier {
-        row![
-            container(rule::horizontal(1))
-                .width(Length::Fill)
-                .padding(padding::right(6)),
-            text("backlog")
-                .size(divider_font_size)
-                .style(theme::text::secondary)
-                .font_maybe(theme::font_style::secondary(theme).map(font::get)),
-            container(rule::horizontal(1))
-                .width(Length::Fill)
-                .padding(padding::left(6))
-        ]
-        .padding(2)
-        .align_y(iced::Alignment::Center)
-    } else {
-        row![]
-    };
+    let divider =
+        if show_backlog_divier {
+            if config.buffer.backlog_separator.text.is_empty() {
+                row![
```

Comment:
With it as a enum we could use a match here

```rust
let row = config.buffer.backlog_separator.text {
    Some(backlog) => row![ rule.width(fill), text(backlog), rule.width(fill) ],
    None => row![ rule.width(fill) ],
};
```

or combine everything into a single `row![]` since you can pass optionals to it.
so you could pass: rule, backlog_text.map(...), backlog_text.map(rule).